src/main/java/Wolox/training/models/Book.java

```diff
@@ -0,0 +1,120 @@
+package Wolox.training.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
When we use PostgreSQL, the best strategy is **sequence**, so Hibernate can use a sequence table optimized by PostgreSQL. It is like declaring the column as **SERIAL** in the DDL script.
```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```